tree/ntupleutil/v7/inc/ROOT/RNTupleImporter.hxx

```diff
@@ -226,6 +229,9 @@ private:
    std::unique_ptr<RNTupleModel> fModel;
    std::unique_ptr<REntry> fEntry;
 
+   static RResult<std::unique_ptr<RNTupleImporter>>
```

Comment:
You can make this a private, non-static method. This way you don't need to explicitly pass the importer, and you can then return an `RResult<void>` to capture any exceptions that may occur.